backend/users/views.py

```diff
@@ -243,3 +243,38 @@ def change_password_view(request):
         return render(request, "users/change_password.html", {"error": error})
 
     return render(request, "users/change_password.html")
+
+
+##############
+##### Frineds Methods
+##############
+
+
+@jwt_login_required
+def add_friend(request, user_id):
+    friend = get_object_or_404(CustomUser, id=user_id)
```

Comment:
what happens in the case of 404

Reply:
don't we have default 404 page? This function on django as I understand helps with detecting the error and by default it will redirect to 404 page if it fails

Reply:
we do teh default 404 only works with django running in production mode